### ISSUE TITLE
[FEATURE REQUEST] Screen layout with automatic switching

### ISSUE THREAD

**Issue by TL1882** [StapleButter/melonDS#2196]
An alternate screen layout that automatically switches to whichever screen has 3D content displayed on it, this would make some games fit perfectly on one standard wide monitor.

**Comment by SirYodaJedi** [comment]
Good idea (would work great with NSMB), but would need a warning saying not to use it in games that have 3D visuals on both screens "at the same time", since it would rapidly flash between the two screens at 60Hz.

**Comment by RSDuck** [comment]
There is already an option to do exactly this under Screen Sizing -> Auto. It also detects dual screen 3D and instead shows both screens evenly.

**Comment by SirYodaJedi** [comment]
> There is already an option to do exactly this under Screen Sizing -> Auto. It also detects dual screen 3D and instead shows both screens evenly.

I see. Doesn't seem to work with Hybrid layout, though, unfortunately.

**Comment by sandwichwater** [comment]
it also doesnt seem to work when the screen emphasis is on top/bottom only
